src/daemon/daemon-client.cpp

```diff
@@ -130,8 +131,9 @@ std::unique_ptr<Gesture> DaemonClient::makeGestureFromSignalParams(
                 "(uudiut)", &type, &direction, &percentage, &fingers,
                 &deviceType, &elapsedTime);
 
-  // std::cout << "GestureType: " << gestureTypeToStr(gestureType) << std::endl;
-  // std::cout << "GestureDirection: " <<
+  // TODO: Jose, update these to use the Logger if you ever decide to use them
```

Comment:
It is safe to remove this comments, they should not be there actually :sweat_smile:

Reply:
done!